### DISCUSSION TITLE
How to exactly know why a GoPro shuts down and reboot itself?

### DISCUSSION THREAD

**Discussion by sanj3k** [gopro/labs#767] [General]
As title says i have more experienced like 5 times that the gopro11 would just record in the 40-50sec range which is like the time of hitting record, putting the drone on the ground and starting the flight. In flight it didn't record already anymore. And when it rebooted i was flying so when i landed i have only seen it. I then started recoding and waited as long as possible but unfortunately after the reboot i didn't get a message why it did this. The assumption is that it overheated but a confirmation would be really good. 

File wise i tried to look around and couldn't find a log file on the SD card. There is just one qrlog but there was nothing related to reboot in it. Do i have any other options to know for sure what's the matter?

**Comment by dnewman-gpsw** [comment]
Factory reset the camera (!FRESET), test it while not in the air. Maybe Labs reset it (!RESET), just set the minimum Labs extensions you are using, like BITR, LOGB etc. Rebooting for overheating at 40-50 seconds seems very unlikely, unless it is naked and poorly cooled (no heat sync, or no fan, or bad naked hack.) Test !MDBGL=1 for more debug output to qrlog.txt.  If the camera reboot for a failure, the log will list the reboot reason (sort of.)  You could also run a script to log when it is recording. ```>r0"recording"~"not"!1N!R```  If recoding stops, without a reboot, that is more likely a bad SD Card or too high a bitrate.

**Reply by sanj3k** [reply]
Thanks a lot will try to produce asap some logs. 

Yes it is a naked. It is with the fan and before that one I used a long time naked 10 without any issues. Only real difference is that on this one i kept front screen which now block airflow. That's why I had this second question for histogram on quik app since it has helped me a lot with nailing exposure to the best. 

Edit: First log with only mdbgl
www.pastebin.com/P1Ad69mF

Edit2: Second log I ran date time qr before. Does this one help you to understand what goes wrong to be honest I didn't get it 
www.pastebin.com/myNZtz05

Edit3: what's strange is that in the root dir of the SD card there is a new created mdb_11.log file but unfortunately it's empty

**Reply by dnewman-gpsw** [reply]
What firmware version is this? As it does appear to be current.

HERO11 should report it version in the log

This is mine:
[2024-02-12 00:59:51] Labs:Main Start 
[2024-02-12 00:59:51] Labs:H22.01.**02.20.70** 
[2024-02-12 00:59:51] Labs:Initialize 

Based on the commands it seems to be supporting it is either 02.10.70 or 02.12.70, you should update this camera (From https://gopro.github.io/labs/)

Update, then do a !FRESET.  Test again.

Unfortunately there is good chance the camera is not 100%, and it is crashing with 
[2024-02-09 21:30:18] wake reason:**HEARTBEAT_TIMEOUT**  

This means the camera froze, and a deadman switch kicked in to reboot the camera.

Not many cameras can do 200 Mbits/s reliably, particularly if you using 60fps or higher.  I run my naked cameras at 170, and I'm only using 24fps. 160Mb/s is pretty safe at all frame rates for most hardware and SD cards. I did a 4k 60fps 8:7 test on my HERO11 at 200Mb/s, and it did not crash/reboot, but it did fail to capture beyond 30s, as expected.

I don't believe heat is the problem, as the GP2 was only 51C at the crash, not very hot at all.  

Run some tests at 150, after you have done the update and the factory reset.

**Reply by sanj3k** [reply]
Yeah you are right I can update it. And yes towards the 50ish degC shouldn't be the problem. Towards the 200mbits i can only say i run 200 forever and never had any issues even though I heard other people have issues. I use the black pro sandisks. With the naked 10 i would experience a frozen led on the Bec board means it either stuck at glowing or stuck at shut off but never had any missing footage after hitting stop button anyway. 

Still i will do the update and resets just to be sure

**Reply by dnewman-gpsw** [reply]
The hardware is only tested and rated to 120Mb/s, so 200Mb/s will fail on many cameras, particularly at higher frame rates. It is also image dependent, as flying through a forest (detail everywhere) with NR01=1 will fail on most cameras.  So successful shooting at 200 is just lucky (the camera will rate control, so you likely didn't get 200, look at you captured bitrate,) but that luck may have run out.  Try at a lower bitrate.

**Reply by sanj3k** [reply]
I understand that. I will still try out and give me a couple of hours i will try to provide a non naked GP11 footage which should be in the 200s.

As i have in 1-2hrs to do all the updates to the naked one: what do you explicitly mean it's not 100% just out of curiosity can you elaborate on that one?

**Reply by dnewman-gpsw** [reply]
Re: not 100% - If the camera overheated to a point that did damage to components, it might never be stable again.  A full camera is well protected against such event (large heat sync prevent thermal runaways), a naked camera is not so protected.

**Reply by sanj3k** [reply]
Here is a log file to updated naked https://pastebin.com/Qphukw4m again heartbeat issue 

At least what I can tell as a difference is that when I ran 200mbits on the naked10 there was no NR01, WIDE, LOGB available it was just bitr200. 

But funny enough I have on the exact same sdcard from that naked11 where it worked once 200mbits 5.3k in the dark dont know if its less stress then but if you are interested i could share the file otherwise here is the metadata reader results: 

![image](https://github.com/gopro/labs/assets/21160640/0e0198b2-e7df-42ee-95a2-6817acc504fe)

`DEVC 
  DVID 1
  DVNM Global Settings
  VERS 8, 2, 2
  FMWR H22.01.02.10.70
  LINF LSU2052503402020
  CINF -42, 52, 35, 116, -115, -95, -108, -98, -121, -109, 42, 54, -22, -86, -43, -40
  CASN C3471324515284
  MINF HERO11 Black
  MUID 1948464342, -1634426483, 908759943, -657085718, 0, 0, 0, 0
  CMOD 12
  MTYP 0
  OREN U
  DZOM Y
  DZST 0
  SMTR N
  PRTN Y
  PTWB 4500K
  PTSH LOW
  PTCL FLAT
  EXPT 1_50SEC
  PIMX 800
  PIMN 800
  PTEV 0.0
  RATE .empty.
  SROT 24.0000
  EISE N
  EISA N/A
  HCTL Off
  AUPT N
  APTO OFF
  AUDO WIND
  BROD .empty.
  BRID 0
  PVUL F
  PRJT GPRO
  SOFF 0
  CLKS 2
  CDAT 0x00000000657E1346
  SCTM 0
  PRNA 18
  PRNU 1
  SCAP N
  CDTM 0
  DUST NO_LIMIT
  VRES 5312, 4648
  VFPS 25000, 1000
  HSGT OFF
  BITR HIGH
  MMOD STEREO
  RAMP .empty.
  TZON 60
  CLKC 0
  DZMX 1.4000
  CTRL Pro
  PWPR PERFORMANCE
  ORDP Y
  CLDP Y
  PIMD MANUAL
DEVC 
  DVID FOVL
  DVNM Large FOV
  ABSC 0.0000
  ZFOV 156.7230
  VFOV W
  MXCF x1
  MAPX 1.0000
  MYCF y1
  MAPY 1.0000
  PYCF r0r1r2r3r4r5r6
  POLY 0.0000, 2.1112, 0.1433, -1.0301, 0.6410, -0.0000, 0.0000
  ZMPL 0.7115
  ARUW 1.1429
  ARWA 1.1429
DEVC 
  DVID USRM
  DVNM User Metadata
  STRM 
    BITR 200
    TUSB 1
    HIST 5
    HSTP -6, 40, 80
    SYID 0
    SYTM 230418080624.12
    LOGB 400
    WIDE 1
    NR01 1
    WAKE 2
    PRXY 0
    WBLK 0
    BOOT 
    USRD 1
DEVC 
  DVID HLMT
  DVNM Highlights
`

**Reply by dnewman-gpsw** [reply]
25fps is more reliable to get close to 200Mb/s.  
As camera at 150Mb/s the camera is still failing, and sssuming you have done factory and Labs resets, the camera seems to be faulty or damaged.

**Reply by sanj3k** [reply]
Thanks a lot for the clarification yes I have done both resets. Will disassemble it tomorrow then have a look

**Comment by ellielo** [comment]
You mentioned checking the QRLOG file — that’s usually the only onboard log accessible to users. If that doesn't show useful info, there's not much else you can extract directly from the SD card. This article might help clarify potential causes like overheating, power draw, or firmware quirks: [https://www.linkedin.com/pulse/gopro-shuts-off-while-recording-macgasm-oxbie](https://www.linkedin.com/pulse/gopro-shuts-off-while-recording-macgasm-oxbie). It also suggests ways to prevent sudden shutdowns. Hopefully, it gives you better insight or a path to more stable recording sessions.
